### DISCUSSION TITLE
Can you create unit tests for webpack configurations?

### DISCUSSION THREAD

**Discussion by Jakobud** [webpack/webpack#13003] [Q&A]
I have a fairly complex webpack configuration for a big project. Is it possible to create some sort of unit tests for my webpack build process? Something that tests all possible scenarios of different types of files that get thrown at webpack and ensure that the proper files get built?

When I try to search for examples of how to do something like this I only find examples of people using webpack to run unit tests for their projects.

**Comment by anshumanv** [comment]
I'm not sure if it's possible to test different types of files,  webpack will throw error during build time anyways if there is a problem with some file. 

We do have a command in webpack-cli to validate config https://webpack.js.org/api/cli/#configtest but I don't think it will help in your use case.
